projects/storybook/src/stories/Header.stories.ts

```
@@ -0,0 +1,48 @@
+import type { Meta, StoryObj } from '@storybook/vue3';
+
+import MyHeader from './Header.vue';
+
+const meta = {
+  /* 👇 The title prop is optional.
+   * See https://storybook.js.org/docs/configure/#configure-story-loading
+   * to learn how to generate automatic titles
+   */
+  title: 'Example/Header',
+  component: MyHeader,
+  render: (args: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)